.buildkite/pipeline-resource-definitions/kibana-gen-ai-daily.yml

```diff
@@ -0,0 +1,54 @@
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
```

Comment:
make sure to add a reference to this file in `locations.yml` - or use the `fix-locations.ts` script